src/server/scripts/Northrend/Naxxramas/boss_loatheb.cpp

```diff
@@ -126,24 +140,39 @@ class boss_loatheb : public CreatureScript
                     me->CastSpell(me, SPELL_SUMMON_SPORE, true);
                     events.RepeatEvent(35000);
                     break;
-                case EVENT_SPELL_NECROTIC_AURA:
+                case EVENT_NECROTIC_AURA:
                     me->CastSpell(me, SPELL_NECROTIC_AURA, true);
+                    Talk(SAY_NECROTIC_AURA_APPLIED);
+                    events.ScheduleEvent(EVENT_NECROTIC_AURA_FADING, 14000);
+                    events.ScheduleEvent(EVENT_NECROTIC_AURA_REMOVED, 17000);
                     events.RepeatEvent(20000);
                     break;
-                case EVENT_SPELL_DEATHBLOOM:
+                case EVENT_DEATHBLOOM:
                     me->CastSpell(me, RAID_MODE(SPELL_DEATHBLOOM_10, SPELL_DEATHBLOOM_25), false);
                     events.RepeatEvent(30000);
                     break;
-                case EVENT_SPELL_INEVITABLE_DOOM:
+                case EVENT_INEVITABLE_DOOM:
                     me->CastSpell(me, RAID_MODE(SPELL_INEVITABLE_DOOM_10, SPELL_INEVITABLE_DOOM_25), false);
-                    events.RepeatEvent(events.GetTimer() < 5 * MINUTE * IN_MILLISECONDS ? 30000 : 15000);
+                    doomCounter++;
+                    if (doomCounter < 6)
+                    {
+                        events.RepeatEvent(30000);
+                    }
+                    else
+                    {
+                        events.RepeatEvent(15000);
+                    }
```

Comment:
```suggestion
                    events.RepeatEvent(doomCounter < 6 ? 30000 : 15000);
```
not sure what is the best code style